web/pwa/components/GeolocPermissionInfoModal.js

```diff
@@ -0,0 +1,190 @@
+    },
+    facilitateText: {
+      color: theme.palette.primary.contrastText,
+      fontWeight: "bold"
+    },
+    locationExplanation: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      justifyContent: "center",
+      textAlign: "center",
+      marginTop: theme.spacing(4),
+      paddingRight: theme.spacing(3),
+      paddingLeft: theme.spacing(3)
+    },
+    alertInfo: {
+      backgroundColor: "#709DFA",
+      margin: theme.spacing(3),
+      marginTop: "auto"
+    },
+    informationText: {
+      color: theme.palette.primary.contrastText,
+      fontStyle: "italic",
+      fontSize: "small"
+    },
+    actionButton: {
+      alignItems: "center",
+      justifyContent: "center",
+      marginBottom: theme.spacing(4),
+      marginTop: theme.spacing(5),
+      display: "flex",
+      flexDirection: "column",
+      flexShrink: "0"
+    },
+    ctaButton: {
+      backgroundColor: theme.palette.background.paper,
+      color: theme.palette.primary.main,
+      "&:hover": {
+        backgroundColor: theme.palette.background.default
+      },
+      marginTop: theme.spacing(2)
+    },
+    subButton: {
+      color: theme.palette.primary.contrastText
+    }
+  }));
+
+  const Transition = React.forwardRef(function Transition(props, ref) {
+    return <Slide direction="up" ref={ref} {...props} />;
+  });
+  const classes = useStyles();
+  const api = useApi();
+  const store = useStoreSyncedWithLocalStorage();
+
+  async function disableWarning() {
+    try {
+      await store.setUserInfo({
+        ...store.userInfo(),
+        disabledWarnings: store
+          .userInfo()
+          .disabledWarnings.concat([
+            DISMISSABLE_WARNINGS.EMPLOYEE_GEOLOCATION_INFORMATION
+          ])
+      });
+      await api.graphQlMutate(
+        DISABLE_WARNING_MUTATION,
+        { warningName: DISMISSABLE_WARNINGS.EMPLOYEE_GEOLOCATION_INFORMATION },
+        { context: { nonPublicApi: true } }
+      );
+    } catch (err) {
+      captureSentryException(err);
+    }
+  }
+
+  const dismissModal = async () => {
+    await disableWarning();
+    handleClose();
+  };
+
+  const acceptGeoLocation = async () => {
+    await disableWarning();
+    askCurrentPosition();
+    handleClose();
+  };
+
+  return (
+    <Dialog
+      open={open}
+      onClose={dismissModal}
+      TransitionComponent={Transition}
+      PaperProps={{ className: `scrollable` }}
+      fullScreen
+    >
+      <Container
+        maxWidth="sm"
+        className={classes.container}
+        disableGutters
+        data-testid="geoloc-modal-container"
+      >
+        <IconButton
+          aria-label="Fermer"
+          className={classes.closeButton}
+          onClick={dismissModal}
+        >
+          <CloseIcon />
+        </IconButton>
+        <Box className={classes.locationExplanation}>
+          <Typography className={classes.sharePositionText}>
+            👋 Partagez votre position géographique
```

Comment:
Maybe this will be done in another ticket, but emojis should be put in <span> I think

Reply:
Ok I've done it for this one.
I'll do it for the next ones I'll put in the code, and we can "fix" existing ones in a new ticket, or as we find them in other tickets.